### PR TITLE
4992 expiring items report is not translated

### LINKED ISSUE #4992: Expiring items report is not translated
## What went wrong? 😲

The Expiring items report is not being translated, this should be in French:

Screenshot 2024-10-04 at 11 59 35 AM

Expiring Items report - no translation keys:
https://github.com/msupply-foundation/open-msupply/blob/bc198972c9bd8fd6c69e0d504e0bd48907997c64/server/reports/expiring-items/2_3_0/src/template.html#L6-L14

## Expected behaviour 🤔

The columns we have translations for should translate

## How to Reproduce 🔨

Steps to reproduce the behaviour:

1. Go to Reports
2. Switch to french/other language
3. Generate expiring items
4. See column headers - all in English

## Your environment 🌱

- Open mSupply Version: 2.3.0 RC5
- Legacy mSupply Central Server Version:

- Platform:

- Database type:

### PR DESCRIPTION
**Pull request by roxy-dao**
Fixes #4992

# 👩🏻‍💻 What does this PR do?
Adds translations for expiring items and stock status - months. (They don't have translations in their respective languages yet so might not see any changes!)

# 🧪 Testing

- [ ] Change language
- [ ] Go to expiring reports
- [ ] See name has changed languages (only one right now 👀 )

# 📃 Documentation

- [ ] **Part of an epic**: documentation will be completed for the feature as a whole
- [x] **No documentation required**: no user facing changes or a bug fix which isn't a change in behaviour
- [ ] **These areas should be updated or checked**: 
  1.
  2.

### CODE REVIEW

File: client/packages/common/src/intl/locales/en/common.json [hunk 1]
```diff
@@ -577,16 +599,87 @@
   "placeholder.search-by-name-or-code": "Search by name or code",
   "placeholder.search-by-notes": "Search by notes",
   "preferences": "Preferences",
+  "report.actual": "Actual",
```
Comment (roxy-dao) [inline client/packages/common/src/intl/locales/en/common.json:602]: I accidentally sorted in my attempt to find dups... 👀

File: client/packages/common/src/intl/locales/en/common.json [hunk 2]
```diff
@@ -312,6 +323,7 @@
   "label.label": "Label",
   "label.last-name": "Last name",
   "label.last-reading": "Last reading",
+  "label.leads-time": "Months leads time",
```
Comment (lache-melvin) [inline client/packages/common/src/intl/locales/en/common.json:326]: Think this is just a sorting change but shouldn't this be lead time 😢

File: client/packages/common/src/intl/locales/en/common.json [hunk 3]
```diff
@@ -577,16 +599,87 @@
...
+  "report.stock-take-variance-report": "Stock Take Variance Report",
+  "report.stocktake-number": "Stocktake number",
+  "report.stocktake-sheet": "Stocktake sheet",
+  "report.suggested-qty": "Suggested Qty",
+  "report.supplied-to": "Supplied to",
+  "report.target-qty": "Target Qty",
+  "report.their-ref": "Their ref",
+  "report.understocked": "Understocked",
+  "report.usage-2-months-prior": "Monthly usage (2 months ago",
+  "report.usage-last-month": "Monthly usage (last month",
```
Comment (lache-melvin) [inline client/packages/common/src/intl/locales/en/common.json:668]: Do you reckon you could add the closing `)` on to these (there are a few, screenshot in issue) as part of this? 🙏

Comment (roxy-dao) [reply client/packages/common/src/intl/locales/en/common.json]: Opps sorry, missed a few

**Review by lache-melvin — APPROVED** [review]
Love you work, thank you!!

**Pull request merged**